keyboards/ergodox_ez/keymaps/ifohancroft/keymap.c

```diff
@@ -195,5 +111,41 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
                                                                               _______,            _______,
                                                           _______,  _______,  _______,            _______,  _______,  _______
     )
+};
 
+    /* Swap Hands
+    * *-------------------------------------------------*
+    * | Del     | \       | '       |         |         |
+    * |---------+---------+---------+---------+---------+---------*
+    * | 6       | Y       | H       | N       |         |         |
+    * |---------+---------+---------+---------+---------+---------|
+    * | 7       | U       | J       | M       |         |         |
+    * |---------+---------+---------+---------+---------+---------|
+    * | 8       | I       | K       | ,       |         | Space   |
+    * |---------+---------+---------+---------+---------+---------|
+    * | 9       | O       | L       | .       |         |         |
+    * |---------+---------+---------+---------+---------+---------|
+    * | 0       | P       | ;       | /       |         |         |
+    * |---------+---------+---------+---------|         |---------|
+    * | =       | ]       |         |         |         |         |
+    * *---------------------------------------*         *---------*
+    */
+
+const keypos_t PROGMEM hand_swap_config[MATRIX_ROWS][MATRIX_COLS] = {
```

Comment:
Unless I am misunderstanding the feature and QMK's documentation on it, you need to configure it yourself to get the keys you want where you want them, just like a keymap. Also, I had no idea ErgoDox EZ has a hand swap config already and I didn't know it can be inherited, but chances are their default configuration doesn't match the one I want.

Please correct me if I am wrong.

Reply:
It only needs to be defined once, and yes, it is inherited.

Reply:
Ah! Thanks!
So how do I set which keys I want to turn into which keys without defining it again? I mean I want a specific swap hand configuration - http://www.keyboard-layout-editor.com/#/gists/4187e40f4c83fc00f0757b234a1ce6cc (scroll down to Swap Hand):

- Right hand stays as it is
- The empty keys on the left hand stay as they are
- The rest of the keys on the left hand turn into what the KLE shows (based on how my right hand is in the Default Layer at the top)